src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -53,12 +60,21 @@ public SonarQubeIssuesReport extractIssueReport(ProjectIssues projectIssues, Inp
   /**
    * Post SQ analysis overview on Stash
    */
-  public void postAnalysisOverview(String project, String repository, String pullRequestId, String sonarQubeURL, int issueThreshold, SonarQubeIssuesReport issueReport, StashClient stashClient){
+  public void postAnalysisOverview(String[] idCard, int issueThreshold, SonarQubeIssuesReport issueReport,
+                                   CoverageIssuesReport coverageReport, StashClient stashClient){
+
+    // Splitting the idCard into variables
+    String project       = idCard[0];
+    String repository    = idCard[1];
+    String pullRequestId = idCard[2];
+    String sonarQubeURL  = idCard[3];
+    String stashURL      = idCard[4];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "stashURL". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "stashURL" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)